modules/vulnerability-management20-send-reports.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_content-type: PROCEDURE
+[id="vulnerability-management20-send-reports_{context}"]
+= Sending vulnerability reports on-demand
+
+[role="_abstract"]
+You can send vulnerability reports immediately, rather than waiting for the scheduled send time.
+
+.Procedure
+. In the {product-title-short} web portal, navigate to *Vulnerability Management 2.0* -> *Vulnerability Reporting* and locate the report you want to send in the list of reports.
```

Comment:
...and locate the report config for which you want to get a report in the list of report configs.